dv/uvm/core_ibex/Makefile

```diff
@@ -135,7 +135,6 @@ $(gen-dirs): %:
 # variable to P in Makefile syntax, but with "last-X-" prepended. At the start
 # of the file, we also define last-X-vars-loaded to 1. You can use this to
 # check whether there was a dump file at all.
-#
```

Comment:
This is a nit, but deleting this line messes up the paragraph structure in the comments. Either please don't do that, or reflow the comments (so that "Note" would be on the same line as "all").

Reply:
Ah, not quite sure why I deleted this line! It has been inserted back into the Makefile.